edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.CloudProxy/CloudConnection.cs

```diff
@@ -172,39 +166,16 @@ enum EventIds
                 TransportConnected
             }
 
-            public static void AttemptingConnectionWithTransport(ITransportSettings[] transportSettings, IIdentity identity, Option<string> modelId)
+            public static void AttemptingConnectionWithTransport(IotHubClientTransportSettings transportSettings, IIdentity identity, Option<string> modelId)
             {
-                string transportType = transportSettings.Length == 1
-                    ? TransportName(transportSettings[0].GetTransportType())
-                    : transportSettings.Select(t => TransportName(t.GetTransportType())).Join("/");
-                string message = $"Attempting to connect to IoT Hub for client {identity.Id} via {transportType}";
+                string message = $"Attempting to connect to IoT Hub for client {identity.Id} via {transportSettings.GetType()}";
                 string withModelIdMessage = modelId.Match(m => $" with modelId {m}", () => string.Empty);
                 Log.LogInformation((int)EventIds.AttemptingTransport, $"{message}{withModelIdMessage}...");
             }
 
-            public static void CreateDeviceClientSuccess(ITransportSettings[] transportSettings, TimeSpan timeout, IIdentity identity)
+            public static void CreateDeviceClientSuccess(IotHubClientTransportSettings transportSettings, TimeSpan timeout, IIdentity identity)
             {
-                string transportType = transportSettings.Length == 1
-                    ? TransportName(transportSettings[0].GetTransportType())
-                    : transportSettings.Select(t => TransportName(t.GetTransportType())).Join("/");
-                Log.LogInformation((int)EventIds.TransportConnected, $"Created cloud proxy for client {identity.Id} via {transportType}, with client operation timeout {timeout.TotalSeconds} seconds.");
-            }
-
-            static string TransportName(TransportType type)
-            {
-                switch (type)
-                {
-                    case TransportType.Amqp_Tcp_Only:
-                        return "AMQP";
-                    case TransportType.Amqp_WebSocket_Only:
-                        return "AMQP over WebSockets";
-                    case TransportType.Mqtt_Tcp_Only:
-                        return "MQTT";
-                    case TransportType.Mqtt_WebSocket_Only:
-                        return "MQTT over WebSockets";
-                    default:
-                        return type.ToString();
-                }
+                Log.LogInformation((int)EventIds.TransportConnected, $"Created cloud proxy for client {identity.Id} via {transportSettings.GetType()}, with client operation timeout {timeout.TotalSeconds} seconds.");
```

Comment:
I think we have [`ToString()`](https://github.com/Azure/azure-iot-sdk-csharp/blob/previews/v2/iothub/device/src/TransportSettings/IotHubClientTransportSettings.cs#L79-L82) to get the transport type with the protocol type together.


```suggestion
                Log.LogInformation((int)EventIds.TransportConnected, $"Created cloud proxy for client {identity.Id} via {transportSettings.ToString()}, with client operation timeout {timeout.TotalSeconds} seconds.");
```